package.json

```diff
@@ -12,7 +12,6 @@
   },
   "dependencies": {
     "axios": "^0.19.2",
-    "cors": "^2.8.5",
```

Comment:
TIL that including this cors library, without configuration, is LESS secure than not using it at all, because by default it adds the `Access-Control-Allow-Origin: *` header.